code/_helpers/game.dm

```diff
@@ -17,14 +17,13 @@
 	return FALSE
 
 /proc/get_area(O)
+	RETURN_TYPE(/area)
 	var/turf/loc = get_turf(O)
-	if(loc)
-		var/area/res = loc.loc
-		.= res
+	return loc?.loc
 
 /proc/get_area_name(N) //get area by its name
```

Comment:
Checking actual uses of this, I'm confused about what it's for.

Reply:
Every use but one appears to be incorrect. I can fix those if you'd like.